.github/workflows/pr-check.yml

```
@@ -15,41 +15,76 @@ jobs:
           cache-dependency-path: backend/pyproject.toml
       - name: Install backend (editable)
+        if: steps.detect.outputs.exists == 'true'
         run: |
           python -m pip install --upgrade pip
           pip install -e backend
       - name: Backend syntax check (compileall)
+        if: steps.detect.outputs.exists == 'true'
         run: python -m compileall -q backend/app
       - name: Backend import smoke test
+        if: steps.detect.outputs.exists == 'true'
         run: python -c "from app.main import app; print('backend app import: ok')"
```

actionlint (1.7.10)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)